components/meter/test/meter-circle.visual-diff.html

```diff
@@ -43,5 +43,15 @@
 	<div class="visual-diff" style="width: 90px;">
 		<d2l-meter-circle dir="rtl" id="completed-rtl" value="5" max="5"></d2l-meter-circle>
 	</div>
+	<!-- Test foreground-light versions-->
+	<div class="visual-diff" style="width: 90px; background-color: darkblue;">
+		<d2l-meter-circle id="no-progress" value="0" max="10" foreground-light></d2l-meter-circle>
+	</div>
+	<div class="visual-diff" style="width: 90px; background-color: darkgreen;">
+		<d2l-meter-circle id="has-progress" value="16" max="47" foreground-light></d2l-meter-circle>
+	</div>
+	<div class="visual-diff" style="width: 90px; background-color: darkred;">
+		<d2l-meter-circle id="completed" value="5" max="5" foreground-light></d2l-meter-circle>
+	</div>
```

Comment:
These should be of the form `no-progress-light`, `has-progress-light` and `completed-light`, just so all the IDs are unique and match what's being tested. And you'll need to add them to the corresponding `.js` file if they're going to get picked up.

Reply:
Ooooh right whoops. Will do